railties/lib/rails/templates/rails/mailers/email.html.erb

```diff
@@ -95,14 +95,16 @@
       </dd>
     <% end %>
 
+    <dt>Format:</dt>
```

Comment:
Always shows Format in order to detect which mime_type is selected for `@email`.